app/controllers/admin/application_controller.rb

```diff
@@ -7,18 +7,12 @@
 module Admin
   class ApplicationController < Administrate::ApplicationController
     before_action :default_params
-    around_action :add_tags_to_logs
 
     def default_params
       resource_params = params.fetch(resource_name, {})
       order = resource_params.fetch(:order, "created_at")
       direction = resource_params.fetch(:direction, "desc")
       params[resource_name] = resource_params.merge(order: order, direction: direction)
     end
-
-    def add_tags_to_logs
```

Comment:
This doesn't seem to be used anywhere

Reply:
yeah, I think this might have lost its usefulness once we migrated to lograge!